### DISCUSSION TITLE
Best way to migrate working install to a totally new host?

### DISCUSSION THREAD

**Discussion by pcm1ke** [teslamate-org/teslamate#2752] [General]
I have an idea of how to accomplish this, but with so many moving parts involved with this project I’m looking for some straight forward advice on how to successfully migrate a completely working installation to a new host without losing any of my historical data. Any help is appreciated!

**Comment by cwanja** [comment]
You would really just need to follow the backup and restore processes on the documents site. https://docs.teslamate.org/docs/maintenance/backup_restore

**Comment by DrMichael** [comment]
IMHO one of the highlights of Teslamate!
